app.ts

```diff
@@ -0,0 +1,292 @@
+let fromParameter = 0;
+let timer: number;
+let recordCount: number;
+
+//// On Window Load
+window.onload = function () {
+	const next = new Next(); // next class
+	const prev = new Prev(); // prev class
+	window.addEventListener("input", debounce(idJump, 500));
+	window.addEventListener("resize", debounce(resizing, 500));
+	getRecordCount()
+		.then(() => {
+			return getHeadings();
+		})
+		.then(() => {
+			return getTable();
+		});
+};
+
+// Gets The 2nd paramater from the 1st parameter
+function getParameters(fromParameter: number): number {
+	let toParameter: number;
+	let noOfRows = getNoOfRows();
+
+	toParameter = fromParameter + noOfRows;
+	return toParameter;
+}
+
+// Gets the number of rows on the screen
+function getNoOfRows(): number {
+	const height = window.innerHeight;
+	let noOfRows = Math.floor(height / 40);
+	return noOfRows;
+}
+
+//// Functions To Create The HTML
+// Heading Row
+function createHeadingRow(headingData: string) {
+	const heading: HTMLElement | null = document.getElementById("heading");
+
+	let headings = `<div class="headings" id="headings">${headingData}</div>`;
+	if (heading !== null) {
+		heading.innerHTML += headings;
+	}
+}
+
+// Table Content
+function createTableContent(contentData: any) {
+	const content: HTMLElement | null = document.getElementById("content");
+
+	let table = `<div id="row-${contentData[0]}" class="rows"></div>`;
+	if (content !== null) {
+		content.innerHTML += table;
+	} else {
+		alert("ERROR");
+	}
```

Comment:
You probably also want to return in this `else` as the rest of your code doesn't really work if the `content` does not exist.